<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

	<arg name="name" 		default="openGo" />
	<arg name="freq" default="25" />

        <node pkg="moticon_insoles" type="sdk_read.py" 	name="moticon_insoles" output="screen">
		<param name="name" 		value="$(arg name)" />
		<param name="left_cop_reference_frame" 			value="human/calcn_l" />
		<param name="right_cop_reference_frame" 		value="human/calcn_r" />
        </node>
	
	<node pkg="imu_complementary_filter" type="complementary_filter_node"
      		name="complementary_filter_gain_node_left" output="screen">
	    <param name="do_bias_estimation" value="true"/>
	    <param name="do_adaptive_gain" value="true"/>
	    <param name="use_mag" value="false"/>
	    <param name="gain_acc" value="0.01"/>
	    <param name="gain_mag" value="0.01"/>
	    <remap from="/imu/data_raw" to="/left/imu_raw" />
	    <remap from="/imu/data" to="/left/imu" />

	</node>
	<node pkg="imu_complementary_filter" type="complementary_filter_node"
      		name="complementary_filter_gain_node_right" output="screen">
	    <param name="do_bias_estimation" value="true"/>
	    <param name="do_adaptive_gain" value="true"/>
	    <param name="use_mag" value="false"/>
	    <param name="gain_acc" value="0.01"/>
	    <param name="gain_mag" value="0.01"/>
	    <remap from="/imu/data_raw" to="/right/imu_raw" />
	    <remap from="/imu/data" to="/right/imu" />

	</node>

<!--node pkg="tf" type="static_transform_publisher" name="left_foot_static_broadcaster" args="-0.3 0 0 0 0.707 0.707 0 map left $(arg freq)" /-->
<!--node pkg="tf" type="static_transform_publisher" name="right_foot_static_broadcaster" args="0.3 0 0 0 0.707 0.707 0 map right $(arg freq)" /-->

<!-- left_foot right_foot  -->

</launch>


